modules/qtwidgets/src/propertylistwidget.cpp

```diff
@@ -195,6 +205,16 @@ bool PropertyListWidget::event(QEvent* e) {
     }
 }
 
+void PropertyListWidget::onProcessorNetworkEvaluationBegin() { 
+    setUpdatesEnabled(false); 
+    listLayout_->setEnabled(false);
+}
```

Comment:
this should take care of most of the issues we encountered so for, shouldn't it? Nice.